docs/tools/CLI/cli-setup/cli-reqs.md

```diff
@@ -228,6 +228,12 @@ Mbed CLI supports a setting for each toolchain path:
 | IAR EWARM Compiler | `C:/Program Files/IAR Systems/Embedded Workbench 8.2/arm/bin/iccarm.exe` | `IAR_PATH` | `C:/Program Files/IAR Systems/Embedded Workbench 8.2/arm`|
 | GCC Arm Embedded Compiler | `/usr/bin/arm-none-eabi-gcc` | `GCC_ARM_PATH` | `/usr/bin`|
 
+##### Using Arm Compiler 6 from Mbed Studio
+Mbed Studio comes with a free version of Arm Compiler 6 when it's used with Mbed OS. It requires configuring the `ARMC6_PATH` as well as setting an environment variable to manage the license.
```

Comment:
What does "when it's used with Mbed OS" mean? Can you use Mbed Studio without Mbed OS? What happens to Arm Compiler 6 then - does it simply refuse to compile?

Reply:
Nope, Mbed Studio's version of Arm Compiler 6 is only free to use when compiling with Mbed OS. Part of it is it won't actually function without Mbed OS, part of it is it part of the free to use license.